tianshou/data/buffer/prio.py

```diff
@@ -85,3 +85,6 @@ def __getitem__(self, index: Union[slice, int, List[int], np.ndarray]) -> Batch:
         batch = super().__getitem__(indice)
         batch.weight = self.get_weight(indice)
         return batch
+
+    def update_beta(self, beta: float) -> None:
+        self._beta = beta
```

Comment:
why?

Reply:
In the Rainbow paper Table 1, it does mention annealing beta from 0.4 to 1.0 . Many reference implementations did it. However, I did some experiments and found little difference, at least on Atari games. So I'm getting rid of it for now.